x-pack/plugins/apm/public/hooks/use_apm_params.ts

```diff
@@ -14,7 +14,7 @@ import { ApmRoutes } from '../components/routing/apm_route_config';
 
 export function useMaybeApmParams<TPath extends PathsOf<ApmRoutes>>(
   path: TPath,
-  optional: true
+  optional = true
```

Comment:
i tried using this function and noticed that the optional param wasn't really optional, so i fixed it here